src/main/webapp/WEB-INF/tags/shared/feedbackSubmissionEdit/feedbackSessionDetailsPanel.tag

```diff
@@ -27,7 +27,7 @@
         <div class="form-group">
           <label class="col-sm-2 control-label">Opening time:</label>
           <div class="col-sm-10">
-            <p class="form-control-static">${feedbackSession.startTimeString}</p>
+            <p class="form-control-static" data-start-time="${feedbackSession.startTimeInIso8601Format}">${feedbackSession.startTimeString}</p>
```

Comment:
Can we standardise the name of the data attribute? It should be the same here as in all the other files.